src/phases/query/IDHack.yml

```diff
@@ -39,3 +81,50 @@ LoadObjectIdTemplate:
         BatchSize: 1000
         Document:
           _id: {^ObjectId: {^FormatString: {"format": "%07d%s", "withArgs": [{^Inc: {start: -2000000, multiplier: 1000000}}, "bbbbbbbbbbbbbbbbb"]}}}
+
+# Prepopulates the collection with documents that have an int ID, field a and field b. Creates
+# indexes on fields a and b.
+LoadIntIdTemplate:
+  Name: {^Parameter: {Name: "Name", Default: "LoadIntIDs"}}
+  Type: Loader
+  Threads: 10
+  Phases:
+    OnlyActiveInPhases:
+      Active: [{^Parameter: {Name: "OnlyActiveInPhase", Default: {unused: "please specify in which phases this actor should be active."}}}]
+      NopInPhasesUpTo: {^Parameter: {Name: "NopInPhasesUpTo", Default: 99}}
+      PhaseConfig:
+        Repeat: 1
+        Database: {^Parameter: {Name: "Database", Default: test}}
+        MultipleThreadsPerCollection: true
+        CollectionCount: 1
+        DocumentCount: {^Parameter: {Name: "DocumentCount", Default: 10000000}}
+        BatchSize: 1000
+        Document:
+          # The multiplier is multiplied by the ActorID, and added to the value.
+          # Since this is the second actor, the ActorIDs increment for each thread starting at 2.
+          # e.g -2000000 + 2 * 1000000 = 0, -1999999 + 2 * 1000000 = 1, etc.
+          # Once complete, the ids in the collection will be the range 0-9999999.
+          _id: {^Inc: {start: -2000000, multiplier: 1000000}}
+          # [10M, 20M)
+          a: {^Inc: {start: -1000000, multiplier: 1000000}}
+          # [20M, 30M)
+          b: {^Inc: {start: 0, multiplier: 1000000}}
+        Indexes:
+          - keys: {a: 1}
+          - keys: {b: 1}
+
+# Creates an identity view (view with an empty pipeline that acts as a no-op) over the given collection.
```

Comment:
If we want this in the future, feels like it deserves it's own little template file. I could see it coming up again and it feels odd to load it via something called "IDHack.yml"

Reply:
Moved to `Views.yml` -- @cswanson310 does that sound good?